main.py

```diff
@@ -2,6 +2,36 @@
 
 from django.db.models import QuerySet
 
+from db.models import Actor, Genre
+
 
 def main() -> QuerySet:
-    pass
+    genres_to_create = ["Western", "Action", "Dramma"]
+    for name in genres_to_create:
+        Genre.objects.create(name=name)
+
+    actors_to_create = [
```

Comment:
assigning variable names should be at the start of function
by that you'll improve code readability and ensure that variables are defined and initialized in a logical order, making it easier to understand and maintain the code